acceptance_tests/__init__.py

```diff
@@ -20,14 +20,18 @@ def web_driver_connection_error(e):
 
 @retry(retry_on_exception=web_driver_connection_error, wait_fixed=1000, stop_max_attempt_number=30)
 def create_browser():
-
-    if os.getenv('HEADLESS', 'True') == 'True':
-        chromedriver_binary.add_chromedriver_to_path()
-        chrome_options = webdriver.ChromeOptions()
-        chrome_options.add_argument("--no-sandbox")
-        return Browser('chrome', headless=True, options=chrome_options)
+    driver_type = os.getenv('WEBDRIVER', 'chrome')
```

Comment:
If we do `os.getenv('WEBDRIVER', 'chrome').lower()` here we can just pass driver_type straight in below rather than passing in chrome or firefox.

Reply:
The drivers seem to have different config options . Changing to driver_type = os.getenv('WEBDRIVER', 'chrome').lower() so we know its state on entry but I think we need to retain different config code

Reply:
It makes no sense to pass in `chrome` or `firefox` as string literals when we can pass in the variable `driver_type`